src/pages/Explore/components/AssetCard.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Flex, Tag, TagLeftIcon, Text, useColorModeValue } from '@chakra-ui/react'
+import type { Asset } from '@shapeshiftoss/types'
+import type { FC } from 'react'
+import { memo, useCallback, useMemo } from 'react'
+import { RiArrowLeftDownLine, RiArrowRightUpLine } from 'react-icons/ri'
+import { useNavigate } from 'react-router-dom'
+
+import { Amount } from '@/components/Amount/Amount'
+import { AssetIcon } from '@/components/AssetIcon'
+import { bnOrZero } from '@/lib/bignumber/bignumber'
+import { selectMarketDataByAssetIdUserCurrency } from '@/state/slices/selectors'
+import { useAppSelector } from '@/state/store'
+
+interface AssetCardProps {
+  asset: Asset
+  width?: string
+}
+
+export const AssetCard: FC<AssetCardProps> = memo(({ asset, width = '80%' }) => {
+  const navigate = useNavigate()
+  const textColor = useColorModeValue('black', 'white')
+
+  const marketData = useAppSelector(state =>
+    selectMarketDataByAssetIdUserCurrency(state, asset.assetId),
+  )
+  const changePercent24Hr = marketData?.changePercent24Hr
+
+  const priceChange = useMemo(() => {
+    if (!changePercent24Hr) return null
+
+    const isPriceChangePositive = bnOrZero(changePercent24Hr).gte(0)
```

Comment:
Same comment as https://github.com/shapeshift/web/pull/10191/files#r2254841239

Reply:
Done in [6f43640](https://github.com/shapeshift/web/pull/10191/commits/6f43640b9aa2cf8b0364dc9b591858409921a466)